README.org

```diff
@@ -12,7 +12,7 @@ Mumbler uses [[http://gradle.org/][Gradle]] as the build system. To build the co
 #+end_src
 
 * Run
-Mumbler requires a special version of the JVM. You can download a [[http://www.oracle.com/technetwork/oracle-labs/program-languages/downloads/index.html][graal JVM]] from Oracle. Mumbler requires version 0.14 or greater.
+Mumbler requires a special version of the JVM. You can download a [[https://www.graalvm.org/downloads/][graal JVM]] from Oracle. Mumbler requires version 19.3.
```

Comment:
is 19.3. a strict requirement? Otherwise I'd add `or newer`.

Reply:
It is not strict I now verified it works with 20. The gradle build specifies 19.3 as a dependency, so I wanted to be hypercorrect, but I agree ~or newer~ shoud be added back.